riscv/src/large_field/runtime.rs

```diff
@@ -311,6 +309,60 @@ impl Runtime {
         self
     }
 
+    fn with_split_vec(mut self) -> Self {
```

Comment:
I think we can add it to the main machine instead of an extra configurable one

Reply:
I have placed it together with poseidon2, since you can not use one without the other.